packages/bsky/src/api/app/bsky/feed/getPostThread.ts

```diff
@@ -118,3 +120,16 @@ type Skeleton = {
   anchor: string
   uris: string[]
 }
+
+const resolveUri = async (ctx: Context, uriStr: string) => {
+  const uri = new AtUri(uriStr)
+  if (uri.host.startsWith('did:')) {
```

Comment:
`getDids` actually already does this check 👍 

this may be a nice function to stick on _something_ so we can reuse it around other routes, but we can always do that once we start scattering it around a bit more